src/forms/ThresholdsSelection.tsx

```diff
@@ -75,6 +77,24 @@ class ThresholdsSelectionInput extends Component<MyProps & InjectedIntlProps, My
             thresholdName: e.target.value
         });
     }
+    sortThresholdValues = (thresholds: Threshold[]) => {
+        let newThresholds;
+        if (this.state.thresholdOrderDirection === 'asc') {
+            this.setState({
+                thresholdOrderDirection: 'dsc'
+            });
+            newThresholds = thresholds.sort((a, b) => a.value - b.value);
+        } else {
+            this.setState({
+                thresholdOrderDirection: 'asc'
+            });
+            newThresholds = thresholds.sort((a, b) => b.value - a.value);
+        };
+        this.props.valueChanged({
+            ...this.props.value,
+            thresholds: newThresholds
+        });
```

Comment:
I think this implements the wrong thing. What the ticket means is that the thresholds should *always be shown* in sorted order (ascending), not that there should be a button to let the user sort them.